server/public/model/config.go

```diff
@@ -2931,6 +2943,30 @@ func (s *DataRetentionSettings) SetDefaults() {
 	}
 }
 
+// GetMessageRetentionTime returns the message retention time as an int.
+// MessageRetentionHours takes precedence over the deprecated MessageRetentionDays.
+func (s *DataRetentionSettings) GetMessageRetentionTime() int {
+	if s.MessageRetentionHours != nil && *s.MessageRetentionHours > 0 {
+		return *s.MessageRetentionHours
+	}
+	if s.MessageRetentionDays != nil {
+		return *s.MessageRetentionDays * 24
+	}
+	return DataRetentionSettingsDefaultMessageRetentionDays * 24
```

Comment:
`MessageRetentionDays` can't be 0 or lower. I think we can even simplify the code here to
```suggestion
	return *s.MessageRetentionDays * 24
```

Reply:
This is no longer valid because I need to check for `*s.MessageRetentionDays > 0`